concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2819,6 +2819,74 @@ jobs:
                     ACCOUNTS_PASSWORD="$(awk '/^secrets_paas_accounts_admin_password:/ { print $2 }' < ../cf-vars-store/cf-vars-store.yml)" \
                       npm run test:acceptance
 
+          - task: ensure-buildpacks-exist
+            config:
+              platform: linux
+              image_resource:
+                type: docker-image
+                source:
+                  repository: governmentpaas/cf-cli
+                  tag: 0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+              inputs:
+                - name: paas-cf
+                - name: cf-vars-store
+              params:
+                CF_ADMIN: admin
+                SYSTEM_DNS_ZONE_NAME: ((system_dns_zone_name))
+              run:
+                path: bash
+                args:
+                  - -c
+                  - |
+                    set -ueo pipefail
+
+                    CF_PASS=$(awk '/cf_admin_password/ {print $2}' cf-vars-store/cf-vars-store.yml | tr -d '"')
+                    API_ENDPOINT="https://api.${SYSTEM_DNS_ZONE_NAME}"
+                    echo | cf login -a "${API_ENDPOINT}" -u "${CF_ADMIN}" -p "${CF_PASS}" > /dev/null
+
+                    SORT="sort"
+
+                    cat <<-BUILDPACKS | "$SORT" > desired_buildpacks.txt
+                    binary_buildpack
+                    staticfile_buildpack
+                    java_buildpack
+                    ruby_buildpack
+                    nodejs_buildpack
+                    go_buildpack
+                    python_buildpack
+                    php_buildpack
+                    BUILDPACKS
```

Comment:
Is there a way we could extract the list of expected buildpacks from the manifest instead of having to hardcode them here? Maybe by extracting the names from the `install_buildpacks` property.

Reply:
IMO the point of this test is that we explicitly have a list of buildpacks that we want.

I would want this test to fail if upstream removed say the `php_buildpack`.

Reply:
There are 2 assertions here.

1. Assert that the buildpacks in the manifest are actually present on the platform
2. Assert that the a specific list of buildpacks are included.

IMO, 2 can be better covered by a manifest unit test, which leaves this test covering the first case to catch issues like the ones we encountered with the package renaming etc.